mdpow/workflows/base.py

```diff
@@ -0,0 +1,166 @@
+# MDPOW: base.py
+# 2022 Cade Duckworth
+
+"""
+:mod:`mdpow.workflows.base` --- Automated workflow base functions
+=================================================================
+
```

Comment:
Add text to explain (primarily for developers) what you need to do to make this module work. Explain (and link) to registry.